tests/components/rfxtrx/test_config_flow.py

```diff
@@ -318,6 +319,33 @@ async def test_import_update(hass):
     assert result["reason"] == "already_configured"
 
 
+async def test_import_migrate(hass):
+    """Test we can import."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+
+    entry = MockConfigEntry(
+        domain=DOMAIN,
+        data={"host": None, "port": None, "device": "/dev/tty123", "debug": False},
+        unique_id=DOMAIN,
+        version=1,
+    )
+    entry.add_to_hass(hass)
+
+    with patch(
+        "homeassistant.config_entries.ConfigEntries.async_reload", return_value=True
+    ):
```

Comment:
this raises some concern, should we be patching HA? what are we preventing?

Reply:
We are preventing the integration to (re)load, no need for that. I thought to patch is as close to the source as possible, but can also patch `async_setup_entry` and `asynd_unload_entry`

Reply:
We normally patch the integration standard api in similar cases, ie `async_setup_entry` and `async_unload_entry`. We can then assert that they were called.